src/sinks/util/service/concurrency.rs

```diff
@@ -1,16 +1,26 @@
 use serde::Serializer;
 use std::fmt;
+use vector_config::configurable_component;
 
 use serde::{
     de::{self, Unexpected, Visitor},
     Deserialize, Deserializer, Serialize,
 };
 
+/// Configuration for outbound request concurrency.
+#[configurable_component(no_ser, no_deser)]
 #[derive(Clone, Copy, Debug, Derivative, Eq, PartialEq)]
 pub enum Concurrency {
+    /// No concurrency allowed.
```

Comment:
Perhaps this should clarify that `Concurrency::None` is equivalent to a `Fixed` concurrency of 1 (If I'm understanding/remembering correctly)?

Reply:
Addressed in 5cb42267b.